c7n/resources/account.py

```diff
@@ -1942,3 +1942,205 @@ def process_account(self, account):
             return False
         account[self.annotation] = list(cross_account)
         return True
+
+
+@actions.register('toggle-config-managed-rule')
```

Comment:
nit: `toggle-managed-rule` might be shorter and just as clear (given that this lives only on the `aws.config-rule` resource)?

Reply:
nevermind, realized this bit lives on account so we need the longer version for clarity :+1: